### DISCUSSION TITLE
What meshlet_triangles refers to ?

### DISCUSSION THREAD

**Discussion by GiuseppeCassisi96** [zeux/meshoptimizer#748] [Q&A] [answered]
Hey, 
what does the `meshlet_triangles `array set by `meshopt_buildMeshlets()` refer to? Does it correspond to specific triangles or to indices in the `meshlet_vertices `array?

Many thanks

**Comment by zeux** [comment, marked as answer]
See https://github.com/zeux/meshoptimizer/blob/6280fd6c8a2a7bece1fe23b0c31cc3ef956655dc/src/meshoptimizer.h#L473-L479

**Reply by GiuseppeCassisi96** [reply]
So, the triangles array contains indices that refers to vertices locally while the vertices contain indices that refers to vertices globally. Is it right?
